src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -28,6 +28,8 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction AcotT = _library.Append(new AcotTableFunction());
         public static readonly TexlFunction AddColumns = _library.Append(new AddColumnsFunction());
         public static readonly TexlFunction And = _library.Append(new VariadicLogicalFunction(isAnd: true));
+        public static readonly TexlFunction Asin = _library.Append(new AsinFunction());
```

Comment:
>_library [](http://example.com/codeflow?start=51&length=8)

These may require a corresponding move from PAClient.

Reply:
Yes, we'll remove it when bringing Power Fx to PAClient; those functions should have been moved before with the rest of the trig functions, it was probably an oversight. It's good that we have a test on PAClient that will fail if there are duplicates, so we won't "forget" to remove by mistake.